README.md

```diff
@@ -14,6 +14,13 @@ Includes:
     built with [Antora](https://antora.org/),
     published at [conforma.dev/docs][conforma-docs]
 
+## Requirements
+
+* Go version >= 1.23
+* hugo (`dnf install hugo`)
+* entr (`dnf install entr`)
+* Must have cloned "ec-cli", "ec-policies", "user-guide", and "enterprise-contract-controller" repositories at the same level as this repo ("conforma.github.io")
+
 ## Live reload preview
 
 To run the preview of the website as changes are made to files within this
```

Comment:
This line suggests running `make preview` if you want to see your local changes reflected on the Conforma site locally. But that doesn't work since `make preview` fetches the content from upstream repos and not the one we have locally.

What worked in my case was running `make antora-local-live` in one terminal and `make hugo-server` in another server. With this, I was able to see my changes in repos getting reflected on the local build of the Conforma site. **Question**: Do I need to mention these 2 commands in this section instead, or am I missing something?